examples/twittermap/web/public/javascripts/map/controllers.js

```diff
@@ -425,6 +425,24 @@ angular.module('cloudberry.map', ['leaflet-directive', 'cloudberry.common','clou
         }
       });
     }
+    
+    //For randomize coordinates by bounding_box
+    var gseed;
```

Comment:
Can we set a different name for this variable? (more meaningful)

Reply:
I changed it to randomizationSeed.